libs/mailroom/ngx/src/lib/modules/list/modal/delete-email-modal.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, Inject } from '@angular/core';
+
+import { ModalRefService, MODAL_DATA } from '@tamu-gisc/ui-kits/ngx/layout/modal';
+
+@Component({
+  selector: 'tamu-gisc-modal',
+  templateUrl: './delete-email-modal.component.html',
+  styleUrls: ['./delete-email-modal.component.scss']
+})
+export class DeleteEmailModalComponent {
+  constructor(@Inject(MODAL_DATA) public readonly data: { message: string }, private readonly modalRef: ModalRefService) {}
+
+  public closeModal() {
+    this.modalRef.close({
+      deleteEmail: true
+    });
+  }
```

Comment:
Should close email only after you get response from server. This close modal immediately if the server sends back a 200 status code regardless of whether the operation resulted in `true` or `false` deletion.

Reply:
Fixed.